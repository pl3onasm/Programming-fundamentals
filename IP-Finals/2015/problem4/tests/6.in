65
3
4
5
6
5
4
6
7
5
7
6
7
8
7
6
5
4
4
3
3
3
3
3
5
2
12
34
45
34
2
34
4
3
6
4
34
5
4
3
7
8
9
87
5
4
3
2
3
5
6
7
12
4
34
6
4
8
6
65
8
6
76
8
6
23