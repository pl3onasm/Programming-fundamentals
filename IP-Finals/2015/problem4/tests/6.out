25,30,47,1,19,20,21,22,23,33,39,46,48,2,6,17,18,32,35,38,45,53,56,3,5,9,16,24,37,44,49,4,7,11,15,34,50,55,58,61,64,8,10,12,14,40,51,13,41,57,60,63,42,26,52,65,27,29,31,36,54,28,59,62,43
